website/static/js/osfHelpers.js

```diff
@@ -701,7 +701,22 @@ var confirmDangerousAction = function (options) {
 
     bootbox.dialog(bootboxOptions);
 };
-
+/**
+ * Maps an object to an array of {key: KEY, value: VALUE} pairs
+ *
+ * @param {Object} obj
+ * @returns {Array} array of key, value pairs
+ **/
+var iterObject = function(obj) {
```

Comment:
Just to be clear that I'm understanding this: So it was already in use in a mako template before the function was defined in this PR? That seems... confusing.